yt_dlp/aes.py

```diff
@@ -124,17 +150,16 @@ def aes_gcm_decrypt_and_verify(data, key, tag, nonce):
         ghash_in = nonce + [0] * fill + bytes_to_intlist((8 * len(nonce)).to_bytes(8, 'big'))
         j0 = ghash(hash_subkey, ghash_in)
 
-    nonce_ctr = j0[:12]
```

Comment:
Yeah, not sure about that, made it based of pycryptodome's implementation (not spec as I should've).
I think that's bc fb_aes_ctr_decrypt does not handle nonce or assumes it's already part of expanded key...